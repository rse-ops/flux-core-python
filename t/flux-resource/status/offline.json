{
  "R": {
    "execution": {
      "R_lite": [
        {
          "children": {
            "core": "0-3"
          },
          "rank": "0-3"
        }
      ],
      "expiration": 0,
      "nodelist": [
        "foo[1-4]"
      ],
      "starttime": 0
    },
    "version": 1
  },
  "drain": {},
  "exclude": "",
  "offline": "1",
  "online": "0,2-3"
}
