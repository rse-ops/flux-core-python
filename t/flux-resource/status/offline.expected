    STATUS NNODES RANKS           NODELIST
     avail      3 0,2-3           foo[1,3-4]
   offline      1 1               foo2
